### PR TITLE
connecting with pages

### LINKED ISSUE #43: connecting navbar with pages

### PR DESCRIPTION
**Pull request by bouboumael**

### CODE REVIEW

**Review by ChristianOlmedo — DISMISSED** [review]

File: src/View/Includes/_navbar.html.twig [hunk 1]
```diff
@@ -6,10 +6,10 @@
...
         <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
             <div class="navbar-nav">
-                <a class="btn-nav" href="">Accueil</a>
-                <a class="btn-nav" href="">Cours</a>
-                <a class="btn-nav" href="">Stages</a>
-		<a class="btn-nav" href="">Contact</a>
+                <a class="btn-nav" href="/haras/haras">Accueil</a>
+                <a class="btn-nav" href="/course/course">Cours</a>
+                <a class="btn-nav" href="/stage/stage">Stages</a>
+                <a class="btn-nav" href="">Contact</a>
```
Comment (LucyMerlier) [inline src/View/Includes/_navbar.html.twig]: "btn-nav" => is it a bootstrap class?

**Review by LucyMerlier — COMMENTED** [review]
"btn-nav" => is it a bootstrap class?

File: src/View/Includes/_navbar.html.twig (hunk 1 above)
Comment (Loic-Code) [reply src/View/Includes/_navbar.html.twig:12]: yes

Comment (bouboumael) [reply src/View/Includes/_navbar.html.twig:12]: yes

Comment (NEL45) [reply src/View/Includes/_navbar.html.twig:12]: yes

**Review by Loic-Code — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**